production/helm/loki/values.yaml

```diff
@@ -902,6 +902,25 @@ singleBinary:
     storageClass: null
     # -- Selector for persistent disk
     selector: null
+  service:
+    # -- Port of the single binary service
+    port: 3100
+    # -- GRPC port of the single binary service
+    grpcPort: 9095
```

Comment:
we intentionally don't expose http and grpc ports in the configuration for the other target services. this allows us to keep the chart simpler with less config. what is the reason for making these configurable? I would much prefer we keep them in line with the others.